### DISCUSSION TITLE
NextJs Image Component - Width in Percentages help

### DISCUSSION THREAD

**Discussion by ll-r-magner** [vercel/next.js#35584] [Help] [answered]
After so much research and googling im stumped. Our navigation contains images that prior to using NextJS Image component were just basically optimized and set to cover a relative container with a width declared as a percentage. However, with NEXT Image I cannot  have a width set in percentages with layout fill and object cover. So currently I have resized the element using media queries which doesn't feel very fluid.

I know that NextJs release 12.1.1 may have a new layout style called "raw" which may fix this issue for me. 

Long question I know but does anyone know a workaround to use layout fill, object cover, and have my container have a width in percentages with NextJs Image component?

**Comment by ll-r-magner** [comment, marked as answer]
I was able to figure this out on my own. Basically the Image component was wrapping the image in a span that had position relative on it. This span was acting as the constraint rather than allowing my container to be the constraint for height/width values. I added some CSS to my container to basically override that span by selected the firstchild and changing the position property back to static(default) using the !important tag just incase as well. Presto it is working as intended.

**Reply by Loque18** [reply]
This works, but it is a bit weird I have to create a css file each time I want to specify the % of the image
